good-enough-practices-for-scientific-computing.tex

```diff
@@ -401,7 +401,7 @@ \section{Software}\label{sec:software}
     Almost all modern text editors provide \emph{tab completion}, so
     that typing the first part of a variable name and then pressing
     the tab key inserts the completed name of the variable.  Employing
-    this means that meaningful variable names are no harder to type
+    this means that meaningful longer variable names are no harder to type
```

Comment:
"meaningful variable names are no harder to type than terse abbreviations" maybe?